cvp_mesh_planner/src/cvp_mesh_planner.cpp

```diff
@@ -85,7 +85,7 @@ uint32_t CVPMeshPlanner::makePlan(const geometry_msgs::msg::PoseStamped& start,
 
   cost = 0;
   float dir_length;
-  if (!path.empty())
+  if (!cancel_planning_ && !path.empty())
```

Comment:
fix that planner returns quickly, when cancelled.
Current backtracking expands the `path` indefinitely, until cancelled. Then, we get to this bit of code.
Without the fix, the loop over the `path` below will take a very, very long time and mbf is unhappy, because the planner thread is not returning.